.env.example

```
@@ -5,6 +5,7 @@ BLOCKSCOUT_BENS_URL="https://bens.services.blockscout.com"
 BLOCKSCOUT_CHAINSCOUT_URL="http://chains.blockscout.com"
 
 BLOCKSCOUT_BS_TIMEOUT=120.0
+BLOCKSCOUT_BS_LIGHT_TIMEOUT=20.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BLOCKSCOUT_BS_LIGHT_TIMEOUT key should go before the BLOCKSCOUT_BS_TIMEOUT key

(UnorderedKey)